### PR TITLE
test: `chainReader` invalid config tests

### PR DESCRIPTION
**Pull request by damiramirez**
### What Changed?
This PR introduces tests to cover edge cases, including:
- Scenarios where the configuration is empty.
- Cases where invalid addresses are provided.

### Reviewer Checklist
- [ ] Code is well-documented
- [ ] Code adheres to Go [naming conventions](https://go.dev/doc/effective_go#names)
- [ ] Code deprecates any old functionality before removing it

### CODE REVIEW

File: chainio/clients/elcontracts/reader_test.go [hunk 1]
```diff
@@ -576,8 +578,19 @@ func TestGetAllocatableMagnitudeAndGetMaxMagnitudes(t *testing.T) {
...
+	op := types.Operator{
+		Address: operatorAddr.String(),
+	}
+
+	operatorDetails, err := chainReader.GetOperatorDetails(ctx, op)
+	assert.NoError(t, err)
+	assert.NotNil(t, operatorDetails)
+	assert.Equal(t, op.Address, operatorDetails.Address)
+	assert.Equal(t, delay, operatorDetails.AllocationDelay)
+	t.Logf("Operator details: %+v", operatorDetails)
```
Comment (maximopalopoli) [inline chainio/clients/elcontracts/reader_test.go]: This log has debugging purposes?

**Review by maximopalopoli — COMMENTED** [review]
LGTM

**Review by pablodeymo — DISMISSED** [review]

File: chainio/clients/elcontracts/reader_test.go (hunk 1 above)
Comment (damiramirez) [reply chainio/clients/elcontracts/reader_test.go]: Done [5b3acac](https://github.com/Layr-Labs/eigensdk-go/pull/447/commits/5b3acace85ebf37b2ec8c8541c5f6c4e1c163921)

**Review by pablodeymo — APPROVED** [review]
